programs/drift/src/instructions/keeper.rs

```diff
@@ -1912,6 +1946,13 @@ pub struct LiquidatePerpPnlForDeposit<'info> {
     pub user_stats: AccountLoader<'info, UserStats>,
 }
 
+#[derive(Accounts)]
+pub struct SetUserStatusToBeingLiquidated<'info> {
+    pub state: Box<Account<'info, State>>,
+    #[account(mut)]
+    pub user: AccountLoader<'info, User>,
```

Comment:
might want to add a signer here as well, i think anchor client is probably just adding it by default on the client side